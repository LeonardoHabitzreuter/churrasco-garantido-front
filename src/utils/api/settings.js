export const BASE_API_URL = process.env.apiHost || 'http://localhost:3001'
