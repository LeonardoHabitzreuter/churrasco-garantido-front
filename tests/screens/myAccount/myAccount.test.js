import React from 'react'
import { shallow } from 'enzyme'
import MyAccount from '../../../src/screens/myAccount'
import api from '../../../src/utils/api'
api.getUser = jest.fn()

describe('MyAccount component', () => {
  test('should renders properly', () => {
    const wrapper = shallow(<MyAccount />)
    expect(wrapper).toMatchSnapshot()
  })
})
