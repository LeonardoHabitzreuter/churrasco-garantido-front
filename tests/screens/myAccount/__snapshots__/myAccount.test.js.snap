// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyAccount component should renders properly 1`] = `
<div
  className="col-sm-8"
>
  <PageHeader
    bsClass="page-header"
  >
    Minha conta
  </PageHeader>
  <Loader
    loading={true}
    size="large"
  />
  <Alert
    handleDismiss={[Function]}
    messages={Array []}
    show={false}
    style=""
  />
  <Form
    buttonName="Atualizar conta"
    fields={
      Array [
        Object {
          "labelName": "Nome",
          "minLength": 6,
          "name": "name",
          "required": true,
          "type": "text",
          "value": "",
        },
        Object {
          "labelName": "Email",
          "name": "email",
          "required": true,
          "type": "email",
          "value": "",
        },
        Object {
          "labelName": "Senha",
          "maxLength": 20,
          "minLength": 6,
          "name": "password",
          "required": true,
          "type": "password",
          "value": "",
        },
        Object {
          "labelName": "Confirme sua senha",
          "maxLength": 20,
          "minLength": 6,
          "name": "confirmPassword",
          "required": true,
          "type": "password",
          "value": "",
        },
      ]
    }
    inputSize={10}
    labelSize={2}
    onSubmit={[Function]}
  />
</div>
`;
